### DISCUSSION TITLE
Set up user roles?

### DISCUSSION THREAD

**Discussion by CancunManny** [blakeblackshear/frigate#14672] [Ask A Question] [answered]
Just by quickly looking around I am guessing the answer is no to setting up user roles in Frigate itself, but figured I might as well ask.  From what I understand Frigate was created with HomeAssistant in mind.  Today a friend asked me about a setup I hadn't thought about.  Her and her neighbors are looking to get cameras setup on their street, only interested in cameras.

If I set up a small server and install Frigate I could also install a docker to run cloudflare, which would allow people to access frigate itself remotely.  However I noticed I can create users in Frigate, but I can't assign roles.  Are the users in Friagate pretty much all or nothing?  If I wanted all users to have ability to log in and see the feeds and recordings, but only some be able to play with the configurations would it be possible?

Assuming that Frigate can't assign roles to users, I could install HomeAssistant for them to access the cameras.  I haven't played with users much in HA.  From what I understand I could set up custom tabs for users.  Using HA would I be able to control which cameras each user can actually see or would it be an all or nothing?  In fact I think setting up which cameras they can see live wouldn't be that much of a problem since I could add specific cams to different users tabs.  But how about the stuff in the media folders?  That would pretty much have to be an all or nothing deal right?

**Comment by NickM-27** [comment, marked as answer]
Currently no but there is a pinned feature request for it.
